workshops/personal_website/README.md

```diff
@@ -163,331 +122,204 @@ Every HTML file follows this template:
+Images are included in HTML via the image tag, or `<img />`. The image tag has
+an attribute called `src`, which will hold the _source_ URL of the image you
+want to display. As an example, if I were to add this picture of Prophet
+Orpheus, I would right click it and get the source URL, which in this case is
+https://github.com/hackclub/dinosaurs/raw/master/smart_dinosaur_docs.png, and
+put it in an image tag like so:
 
-> ![](img/write_html_template.gif)
+```
+<img src="https://github.com/hackclub/dinosaurs/raw/master/smart_dinosaur_docs.png" />
+```
 
-### 3) Adding to the Template
+You may have noticed that the image tag doesn't have a closing tag like
+`<h1></h1>` or `<body></body>`. That's because it is a self-closing tag. In
+fact, the `/` that comes right before `>` is how it does this.
 
-There's a lot going on in the below template. For now, let's just understand
-that there are two sections in this template, the **`head`** and the **`body`**:
+Go ahead and add this into your `index.html` now. I put my picture before my
+heading, and my code looks like this:
 
 ```html
 <!DOCTYPE html>
 <html>
     <head>
-            ← this section is called the "head" of the HTML document
     </head>
     <body>
-            ← this section is called the "body" of the HTML document
+        <img src="https://github.com/hackclub/dinosaurs/raw/master/smart_dinosaur_docs.png"/>
+        <h1>Prophet Orpheus</h1>
+        <p>Coder Dino</p>
+        <p>Will code for food</p>
     </body>
 </html>
 ```
 
-- Everything between the `<head>` and `</head>` tags is called the **head** of
-  the page. `<head>` is considered an "opening tag" because it "opens" the head
-  section. Since `</head>` starts with `</` (instead of just `<`), it is
-  considered a "closing tag" and "closes", or indicates the end of, the head
-  section.
-- Similarly, everything between the `<body>` and `</body>` tags is called the
-  **body** of the page. `<body>` is the opening tag and `</body>` is the closing
-  tag. Together, the opening tag and the closing tag compromise the `body`
-  element.
-- Note: most HTML elements have an _opening tag_ and a _closing tag_. You'll
-  want to get used to this pattern :smile:.
-
-### 4) Opening the Website Preview
-
-Let's see what our website looks like so far:
-
-1. First, save the file by clicking **"File"** _(on the top right)_ → **"Save"**
-   (or use the shortcut **CTRL + S / Command + S**)
-2. Preview what the website looks like by clicking **"Preview"** → **"Live
-   Preview File"**
+Now, your image may either be too big, or too small. Fret not. CSS will allow
+you to manipulate the styling of your page in all your needs.
 
-> ![](img/c9_live_preview.gif)
+## Part III: The CSS File
 
-As you can see, the page is still blank. This is because we haven't added any
-**content** in the **`body`** section yet. Let's add some!
+So what is CSS? CSS is a language used for styling elements, hence its name,
```

Comment:
Rephrase: "CSS, also known as Cascading Style Sheets, is a language used for styling the content on a web page, otherwise known as elements."

We hadn't introduced the word elements to describe the content of the page before this, so a brief explanation should be there.